openshift/productization/generate-dockerfiles/Dockerfile.in

```diff
@@ -4,15 +4,15 @@ COPY . .
 RUN go build -o /tmp/$SUBCOMPONENT ./cmd/$GO_PACKAGE
 
 FROM ubi8:8-released
-COPY --from=builder /tmp/$SUBCOMPONENT /usr/bin/$SUBCOMPONENT
+COPY --from=builder /tmp/$SUBCOMPONENT /ko-app/$SUBCOMPONENT
 
 LABEL \
       com.redhat.component="openshift-serverless-1-tech-preview-$COMPONENT-$SUBCOMPONENT-rhel8-container" \
       name="openshift-serverless-1-tech-preview/$COMPONENT-$SUBCOMPONENT-rhel8" \
-      version="0.9.0" \
+      version="$VERSION" \
       summary="Red Hat OpenShift Serverless 1 $CAPITALIZED_COMPONENT $CAPITALIZED_SUBCOMPONENT" \
-      maintainer="mthoemme@redhat.com" \
+      maintainer="wvanwinc@redhat.com" \
```

Comment:
Should this be `serverless-support@redhat.com`?

Reply:
Per our offline discussion.. I'm looking into whether `maintainer` should be a mailing list. If yes, I'll fix in a subsequent PR. In the interest of getting this PR merged ASAP, I'll leave as is for now.

Reply:
According to Adam Haile, OpenShift generally required the `maintainer` to be a mailing list. With this in mind, we can set it to `serverless-support@redhat.com`. I'll do this in another PR.